### DISCUSSION TITLE
StageWebView social networks oAuth

### DISCUSSION THREAD

**Discussion by MalacTheLittle** [airsdk/Adobe-Runtime-Support#1369] [Q&A]
When trying to login to our site from StageWebView(true) on Android, I'm getting `Authorisation Error
Error 403: disallowed_useragent
You can’t sign in from this screen because this app doesn’t comply with Google’s embedded webview policy. If this app has a website, you can open a web browser and try signing in from there.`

Similar thing happens on Windows with message `This browser or app may not be secure.`

Does anybody know how to solve this problem?

**Comment by ajwfrost** [comment]
Presumably you're coming up against this:
https://developers.googleblog.com/2021/06/upcoming-security-changes-to-googles-oauth-2.0-authorization-endpoint.html

> Embedded webviews implementing or extending Android WebView do not comply with Google's secure browser policy for its OAuth 2.0 Authorization Endpoint. Apps should allow general, third-party links to be handled by the default behaviors of the operating system, enabling a user's preferred routing to their chosen default web browser or another developer's preferred routing to its installed app through Android App Links.

**Comment by MalacTheLittle** [comment]
@ajwfrost Is it possible to set app to fulfill all the requirements for such action?
